test/common/network/address_impl_test.cc

```diff
@@ -1,11 +1,63 @@
 #include "envoy/common/exception.h"
 
 #include "common/network/address_impl.h"
+#include "test/test_common/network_utility.h"
+#include "test/test_common/utility.h"
 
 #include <sys/un.h>
+#include <unistd.h>
+#include <fcntl.h>
 
 namespace Network {
 namespace Address {
+namespace {
+class ScopedFdCloser {
+public:
+  ScopedFdCloser(int fd) : fd_(fd) {}
+  ~ScopedFdCloser() { close(fd_); }
+
+private:
+  int fd_;
+};
+
+void makeFdBlocking(int fd) {
+  const int flags = ::fcntl(fd, F_GETFL, 0);
```

Comment:
Not really necessary at the moment, but it makes it clear you're using something not in the current namespace. I'll fix the close above.